src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectDetails.tsx

```diff
@@ -59,7 +48,7 @@ const ProjectDetails: React.FC<ProjectDetailsProps> = ({ project }) => {
             <b>Start Date:</b> {start}
           </Col>
           <Col className={allColsStyle} sm={4} md={4} lg={3} xl={2}>
-            <b>End Date:</b> {end}
+            <b>End Date:</b> {datePipe(project.endDate)}
```

Comment:
so when i try to run it locally it doesn't compile becuase here you're passing in `date | undefined` into a function that only accepts date. I guess one way to fix that would be that if datepipe gets undefined, return empty string